README.md

```diff
@@ -6,44 +6,38 @@
   <a href="https://grafana.slack.com/archives/C01D981PEE5"><img src="https://img.shields.io/badge/join%20slack-%23tempo-brightgreen.svg" alt="Slack" /></a>
   <a href="https://community.grafana.com/c/grafana-tempo/40"><img src="https://img.shields.io/badge/discuss-tempo%20forum-orange.svg" alt="Community Forum" /></a>
   <a href="https://goreportcard.com/report/github.com/grafana/tempo"><img src="https://goreportcard.com/badge/github.com/grafana/tempo" alt="Go Report Card" /></a>
-  <a href="https://github.com/grafana/tempo/search?q=blerg"><img src="https://img.shields.io/github/search/grafana/tempo/blerg" alt="Blerg Counter" /></a>
 </p>
 
-
-Grafana Tempo is an open source, easy-to-use and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki. Tempo can be used with any of the open source tracing protocols, including Jaeger, Zipkin, OpenCensus, Kafka, and OpenTelemetry.  It supports key/value lookup only and is designed to work in concert with logs and metrics (exemplars) for discovery.
+Grafana Tempo is an open source, easy-to-use and high-scale distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Prometheus, and Loki.
 
 Tempo is Jaeger, Zipkin, Kafka, OpenCensus and OpenTelemetry compatible.  It ingests batches in any of the mentioned formats, buffers them and then writes them to Azure, GCS, S3 or local disk.  As such it is robust, cheap and easy to operate!
 
-<p align="center"><img src="docs/sources/getting-started/assets/grafana-query.png" alt="Tempo Screenshot"></p>
+Tempo implements [TraceQL](https://grafana.com/docs/tempo/latest/traceql/), a traces first query language inspired by LogQL and PromQL. This query language allows users to very precisely and easily select spans and jump directly to the spans fulfilling the specified conditions:
```

Comment:
Suggesting a hyphen assuming that "traces first" means that the query language is centered around traces, and not that this query language is the first QL for traces. Is that the correct meaning? 
```suggestion
Tempo implements [TraceQL](https://grafana.com/docs/tempo/latest/traceql/), a traces-first query language inspired by LogQL and PromQL. This query language allows users to very precisely and easily select spans and jump directly to the spans fulfilling the specified conditions:
```